content/en/docs/setup/pick-right-solution.md

```diff
@@ -52,6 +52,8 @@ a Kubernetes cluster from scratch.
 
 * [Kubeadm-dind](https://github.com/kubernetes-sigs/kubeadm-dind-cluster) is a multi-node (while minikube is single-node) Kubernetes cluster which only requires a docker daemon. It uses docker-in-docker technique to spawn the Kubernetes cluster.
 
+* [Kubernetes IN Docker](https://github.com/kubernetes-sigs/kind) is a tool for running local Kubernetes clusters using Docker container "nodes". It is primarily designed for testing Kubernetes 1.11+, initially targeting the [conformance tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md). You can use it to create multi-control-plane(while kubeadm-dind is single-control-plane) or multi-node Kubernetes cluster.
```

Comment:
best to change the last sentence to:
> You can use it to create multi-node or multi-control-plane Kubernetes clusters.

Reply:
+1, also we can drop the conformance tests note, I've meant to do that upstream. We test other things with kind. `It is primarily designed for testing Kubernetes 1.11+.` is sufficient on it's own, the conformance bit is just noise now.

Reply:
@neolit123 @BenTheElder Thanks. :+1:  I have updated.